### ISSUE TITLE
"TypeError: n.files.has is not a function" when running from webpack 

### ISSUE THREAD

**Issue by MiloCasagrande** [FullHuman/purgecss#539]
After upgrading to `purgecss-webpack-plugin` `3.1.0`, webpack fails with the following error:

```
[webpack-cli] TypeError: n.files.has is not a function
    at /workspace/node_modules/purgecss-webpack-plugin/lib/purgecss-webpack-plugin.js:1:1060
    at Array.filter (<anonymous>)
    at module.exports.runPluginHook (/workspace/node_modules/purgecss-webpack-plugin/lib/purgecss-webpack-plugin.js:1:976)
    at /workspace/node_modules/purgecss-webpack-plugin/lib/purgecss-webpack-plugin.js:1:707
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/workspace/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:13:17)
    at AsyncSeriesHook.lazyCompileHook (/workspace/node_modules/tapable/lib/Hook.js:154:20)
    at /workspace/node_modules/webpack/lib/Compilation.js:1405:32
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/workspace/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at AsyncSeriesHook.lazyCompileHook (/workspace/node_modules/tapable/lib/Hook.js:154:20)
    at Compilation.seal (/workspace/node_modules/webpack/lib/Compilation.js:1342:27)
    at /workspace/node_modules/webpack/lib/Compiler.js:675:18
    at /workspace/node_modules/webpack/lib/Compilation.js:1261:4
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/workspace/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:31:1)
    at AsyncSeriesHook.lazyCompileHook (/workspace/node_modules/tapable/lib/Hook.js:154:20)
    at Compilation.finish (/workspace/node_modules/webpack/lib/Compilation.js:1253:28)
    at /workspace/node_modules/webpack/lib/Compiler.js:672:17
```

It was working correctly with `3.0.0`.

Environment on macOS `10.15.7`:

* webpack: 4.44.2
* webpack-cli: 4.2.0
* postcss: 8.2.1
* mini-css-extract-plugin: 1.3.3
* css-loader: 5.0.1

**Comment by MiloCasagrande** [comment]
I suspect it's this line:

https://github.com/FullHuman/purgecss/blob/master/packages/purgecss-webpack-plugin/src/index.ts#L78 :

```
return chunk.files.has(name);
```

It should probably be:
```
return chunk.files.includes(name);
```

**Comment by Ffloriel** [comment]
Yes, webpack 5 changed the API for `chunk.files`. Can you try version purgecss-webpack-plugin 3.1.1 and let me know if it is working as expected

**Comment by MiloCasagrande** [comment]
Hi,

thanks for the quick reply.
I tried with `3.1.1` but it still gives me the same error unfortunately.

I tried to log the reported webpack version from `lib/purgecss-webpack-plugin.js` and it says `5.10.1`, this is my `npm ls webpack` output:
```
├─┬ purgecss-webpack-plugin@3.1.1
│ └── webpack@5.10.1
└── webpack@4.44.2
```
I suspect it picks up its webpack version at initialization, wild guess though, not sure how the internals work there.
